### DISCUSSION TITLE
I am using Google Meet. Members who have joined the room as publisher and members who have joined the room are not displayed.

### DISCUSSION THREAD

**Discussion by elm-asano** [flutterjanus/flutter_janus_client#171] [Q&A] [answered]
nice to meet you.
We are using janus_client.

I used Google Meet for the Sample project, but although I can see my own broadcast screen, remote members who have joined as publisher are not displayed.

For operation confirmation, we also use Janus' videoroomdemo.

Members who joined via GoogleMeet will be added to Janus' videoroomdemo.
will be displayed.

Members who have joined janus's videodemo will not be displayed on GoogleMeet.

Users who joined GoogleMeet were also not visible to each other.

What could be the reason why publisher members are not displayed on GoogleMeet?

The protocol uses https.
Only the connection destination of the program has been changed.

**Comment by shivanshtalwar** [comment, marked as answer]
Could you share a minimal reproduction

**Reply by elm-asano** [reply]
Hello, thank you for your reply.
I used Google Translate so please forgive me if it is difficult to understand.

Regarding the matter you asked about, I think it was resolved by changing from https to WebSocket.

I am using the example of the provided janus_client project, but even when I run the original version, the response seems to be too slow to be of any use when connecting via https.

Is there a way to make it work with https?

**Reply by shivanshtalwar** [reply]
Ideally it shouldn't be slow unless your network is throttling 
But you can increase
 ```js
int maxEvent = 10
```
In JanusClient constructor and see if that speeds up flow
